### PR TITLE
Move bigquery code into a subpackage

### PR DESCRIPTION
**Pull request by dhermes**
This PR was created via: https://gist.github.com/dhermes/e239691aa584bd56a5352e34aad27cf3

```
export PROJECT_DIR="${HOME}/google-cloud-python"
export READMES_DIR="${HOME}/i-wrote-some-readmes-for-2357"
cd ${PROJECT_DIR}
git worktree add -b make-bigquery-subpackage ../hotfix official/master
python make_commits.py \
  --git-root "${PROJECT_DIR}/../hotfix" \
  --package bigquery \
  --package-name "Google BigQuery" \
  --readme "${READMES_DIR}/bigquery/README.rst"
```

### CODE REVIEW

File: bigquery/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    # Perform a synchronous query.
+    QUERY = (
+        'SELECT name FROM [bigquery-public-data:usa_names.usa_1910_2013] '
+        'WHERE state = "TX"')
+    query = client.run_sync_query('%s LIMIT 100' % QUERY)
+    query.timeout_ms = TIMEOUT_MS
+    query.run()
+
+    for row in query.rows:
+        print row
```
Comment (jonparrott) [inline bigquery/README.rst]: `print(...)`?

**Comment by dhermes** [comment]
Folded the `print(...)` change into 2nd commit

**Comment by tseaver** [comment]
LGTM

**Comment by dhermes** [comment]
As with #2416 and #2417, Pushed to resolve rebase conflict and re-ran script. Won't wait on Travis since a) that could be awhile and b) this will be painful.

**Pull request merged**